po/tr.po

```diff
@@ -51,7 +54,7 @@ msgstr ""
 #. it is relatively easy to type by an adult without making mistakes.
 #: bambam.py:51
 msgid "unmute"
-msgstr ""
+msgstr "sesi aç"
```

Comment:
@sabriunal please change to a translation that does not contain a space.

Reply:
Thanks. I changed it.